### PR TITLE
fix: we did not add a metadata_keywords field to pages

### PR DESCRIPTION
**Pull request by pdpinch**
#### What are the relevant tickets?

no issue, I just noticed that the release notes were misleading

#### What's this PR do?

Corrects the release notes to remove "add ... metadata_keywords fields to pages"

#### How should this be manually tested?

I hope this doesn't mess up Doof too much.

### CODE REVIEW

**Comment by pdpinch** [comment]
@gumaerc when you get a chance?

File: RELEASE.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ Version 0.22.2 (Released June 09, 2022)
 --------------
 
 - plain text description fields should be text and not string (#192)
-- add metadata_description and metadata_keywords fields to pages (#190)
+- add metadata_description field to pages (#190)
```
Comment (gumaerc) [inline RELEASE.rst:8]: Good catch, and in addition to this the field is simply called `description` per Ferdi's recommendation

**Review by gumaerc — CHANGES_REQUESTED** [review]

File: RELEASE.rst (hunk 1 above)
Comment (pdpinch) [reply RELEASE.rst]: updated!

**Review by gumaerc — APPROVED** [review]
:+1:

**Pull request merged**